tests/integration/nm/ovs_test.py

```diff
@@ -0,0 +1,185 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+from contextlib import contextmanager
+
+import pytest
+
+from libnmstate import nm
+from libnmstate.schema import OvsBridge as OB
+
+from .testlib import mainloop
+
+
+BRIDGE0 = 'brtest0'
+ETH1 = 'eth1'
+
+
+@pytest.fixture
+def bridge_minimum_config():
+    return {
+        OB.CONFIG_SUBTREE: {
+            OB.PORT_SUBTREE: []
+        }
+    }
+
+
+@pytest.fixture
+def bridge_default_config():
+    return {
+        OB.CONFIG_SUBTREE: {
+            OB.OPTIONS_SUBTREE: {
+                OB.FAIL_MODE: '',
+                OB.MCAST_SNOOPING_ENABLED: False,
+                OB.RSTP: False,
+                OB.STP: False
+            },
+            OB.PORT_SUBTREE: []
+        }
+    }
+
+
+def test_create_and_remove_minimum_config_bridge(bridge_minimum_config,
+                                                 bridge_default_config):
+    bridge_desired_state = bridge_minimum_config
+
+    with _bridge_interface(bridge_desired_state):
+        bridge_current_state = _get_bridge_current_state()
+        assert bridge_current_state == bridge_default_config
+        assert not bridge_current_state[OB.CONFIG_SUBTREE][OB.PORT_SUBTREE]
```

Comment:
IMHO the assert should be that it is the empty list

Reply:
Done